python/dask_cudf/dask_cudf/tests/test_groupby.py

```diff
@@ -584,7 +584,7 @@ def test_groupby_categorical_key():
 
 
 @pytest.mark.parametrize("as_index", [True, False])
-@pytest.mark.parametrize("split_out", [None, 1, 2])
+@pytest.mark.parametrize("split_out", [1, 2])
```

Comment:
Would it be worth it to retain the original parameters here and omit it from the affected `aggregate` call when it's null? This is roughly similar to what's being done for the `test_groupby_dropna_*` tests in this file with `dropna` and I imagine is being done to ensure we match up with the default behavior of upstream Dask.